Makefile

```diff
@@ -123,7 +124,11 @@ release-build:
 	make release-build-macos
 	make release-build-windows
 
-test-alpine: release-build-alpine
+.PHONY: create-tester-image
+create-tester-image:
+	docker build -t $(ALPINE_TESTER) - < ./Dockerfile.alpine_tester
+
+test-alpine: release-build-alpine create-tester-image
```

Comment:
Oh, nice find. It's possible I never tested that because the release-build-alpine build is so slow.

I guess the reason is that unlike before, $(ALPINE_TESTER) is not always an x86 image anymore. Instead, it's built for arm on arm machines. Will double check.